### PR TITLE
(IAC-1455) - Removal of Inappropriate Terminology

### PR DESCRIPTION
**Pull request by david22swan**
Removal of any offensive or inappropriate terminology from the code base.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+* [Base provider](https://github.com/puppetlabs/puppetlabs-panos/blob/main/lib/puppet/provider/panos_provider.rb)
+* [Actual provider with validation and xml processing](https://github.com/puppetlabs/puppetlabs-panos/blob/main/lib/puppet/provider/panos_address/panos_address.rb)
+* [New unit tests](https://github.com/puppetlabs/puppetlabs-panos/blob/main/spec/unit/puppet/provider/panos_provider_spec.rb) for 100% coverage.
 
 ## [Find the full Resource API documentation here](https://puppet.com/docs/puppet/latest/custom_resources.html)
 
 ## Related Documents
 
-* The [Resource API specs](https://github.com/puppetlabs/puppet-specifications/blob/master/language/resource-api/README.md) describes details of all the capabilities of this gem.
+* The [Resource API specs](https://github.com/puppetlabs/puppet-specifications/blob/masmainter/language/resource-api/README.md) describes details of all the capabilities of this gem.
```
Comment (DavidS) [inline README.md:15]: that repo hasn't change branches yet

File: appveyor.yml [hunk 2]
```diff
@@ -38,4 +38,4 @@ notifications:
 # Uncomment this block to enable RDP access to the AppVeyor test instance for
 # debugging purposes.
 #on_finish:
-#  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+#  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/main/scripts/enable-rdp.ps1'))
```
Comment (DavidS) [inline appveyor.yml:41]: that repo hasn't change branches yet

**Review by DavidS — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 2 above)
Comment (david22swan) [reply appveyor.yml]: reverted

File: README.md (hunk 1 above)
Comment (david22swan) [reply README.md]: reverted

**Comment by david22swan** [comment]
Unable to replicate failures locally

**Comment by david22swan** [comment]
Current travis failures ticketed here and unrelated to PR: https://tickets.puppetlabs.com/browse/IAC-1556

**Review by DavidS — APPROVED** [review]

**Pull request merged**